src/channels/build.ts

```diff
@@ -63,17 +63,21 @@ export class BuildChannel extends BaseChannel {
       `Requesting upload for build ${metadata.filename} (${metadata.type}, ${metadata.size} bytes)`,
     );
 
-    const { url, build_id } = await this.requestUploadUrl(metadata, appSlug);
+    const response = await this.requestUploadUrl(metadata, appSlug);
```

Comment:
Any reason to not do the destructing here?

Reply:
Ah I mistakenly thought this would throw an error, but it looks like it doesn't

```
const result = { url: "foo", build_id: "123" }
const { existing } = result;

console.log(existing)
// undefined
```

I'll use a destructure.